### DISCUSSION TITLE
Welcome to try.elinq Discussions!

### DISCUSSION THREAD

**Discussion by kostat** [streamx-co/xlinq.live#5] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by heckl** [comment]
Hi,
I have found the EFcore nuget package: Streamx.Linq.SQL.EFCore

Do you have a version for regular EF?

Whishes,
István

**Reply by kostat** [reply]
No, unfortunately classic EF is not supported. 

Best,
Kosta
